service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterMapper.java

```diff
@@ -200,4 +206,24 @@ private void setEncounterReasonAndType(Encounter encounter, POCDMT000002UK01Clin
         }
         return Optional.of(patient);
     }
+
+    private void addEncounterType(POCDMT000002UK01Encounter encounterITK, Encounter encounter) {
```

Comment:
I don't think we should populate encounter.type this way
It's a CodeableConcept and it should match the valueset. Let's leave it empty for now - until v2.1 guidance is not ready.